### ISSUE TITLE
Unspecific message 'Server error' when trying to duplicate Repeater Matrix field

### ISSUE THREAD

**Issue by fitzliputz** [ryancramerdesign/ProcessWire#2071]
Full error message is 'Server Error. We're sorry! The server encountered an internal error and was unable to complete your request. Please try again later.'

I suppose, intended was to handle the clone trial like the one to duplicate a core repeater field, where you receive a proper specific message.

**Comment by teppokoivula** [comment]
Hey @fitzliputz. Current repository for ProcessWire issues is located at https://github.com/processwire/processwire-issues – you should reopen this issue there. Please also include the version of ProcessWire in the issue.

The message you mention here is also a generic error message from the server. It would be helpful if you could check server log files (ProcessWire's log files in /site/assets/logs/ and Apache error log, the location of which depends on your server setup) for more detailed information. Thanks!

**Comment by teppokoivula** [comment]
Right, looks like you did open a new issue at https://github.com/processwire/processwire-issues/issues/923. Nice :)

Would probably make sense to close this one to avoid any confusion, though.
